packages/neuron-wallet/tests-e2e/application/index.ts

```diff
@@ -211,4 +212,21 @@ export default class Application {
     }, text)
     log(`setValue - ${selector} = ${result}`);
   }
+
+  getOSplatform():string {
+    let os = require('os');
+    let platform = os.platform();
+    console.log(`current os platform is ${platform}`)
+    return platform;
+  }
+
+  // goto Setting page from menu according to OS platform
+  async gotoSettingPageFromMenu() {
+    if(this.osPlatform.includes("darwin")) {
+      await this.clickMenu(['Electron', 'Preferences...'])
+    } else if(this.osPlatform.includes("win32")) {
```

Comment:
On both Windows and Linux `Settings` is under `Help` so the check for `win32` could be removed.

Reply:
Not a big problem for now.